dotcom-rendering/src/server/handler.footballDataPage.web.ts

```diff
@@ -32,18 +32,32 @@ const decidePageKind = (pageId: string): FootballMatchKind => {
 	throw new Error('Could not determine football page kind');
 };
 
+const regionsPriority = [
+	'Internationals',
+	'English',
+	'European',
+	'Scottish',
+	'Rest of world',
+];
+
+export const sortRegionsFunction = (a: Region, b: Region): number => {
+	return regionsPriority.indexOf(a.name) - regionsPriority.indexOf(b.name);
```

Comment:
Any new regions not in the list would come first (`indexOf` returning -1) which I think is fine